pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -832,6 +833,25 @@ func (asw *actualStateOfWorld) newAttachedVolume(
 	}
 }
 
+func (asw *actualStateOfWorld) requiresRemount(spec *volume.Spec) bool {
```

Comment:
Should this be implemented somewhere inside https://github.com/kubernetes/kubernetes/blob/0c4e1d04dfce877cf8a8e11b62e81eabf9df62a4/pkg/volume/csi/csi_plugin.go.

The logic in this file is not specific to csi.

Reply:
done. RequiresRemount wasn't accepting `volume.Spec`.